docs/advanced-configurations.rst

```diff
@@ -335,6 +335,13 @@ RHEL
 
 Replace `RH_USERNAME` and `RH_PASSWORD` with your Red Hat account username and password.
 
+~~~~~
```

Comment:
why are we using `~~~`

i didnt find it in [1]

[1] https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
It seems to render OK:
https://mellanox.github.io/network-operator-docs/advanced-configurations.html#rhel-specific-build-parameters